### ISSUE TITLE
Helm chart directory is missing index.yaml

### ISSUE THREAD

**Issue by heywood8** [joeferner/redis-commander#447]
The command from README:

`helm repo add redis-commander https://raw.githubusercontent.com/joeferner/redis-commander/master/k8s/helm-chart/`

answers with:

Error: looks like "https://raw.githubusercontent.com/joeferner/redis-commander/master/k8s/helm-chart/" is not a valid chart repository or cannot be reached: failed to fetch https://raw.githubusercontent.com/joeferner/redis-commander/master/k8s/helm-chart/index.yaml : 404 Not Found

This index.yaml was submitted in original Pull request for a helm chart:
https://github.com/joeferner/redis-commander/pull/412/files

**Comment by sseide** [comment]
yes you are right, to just add this a a helm repo a packaged version of this chart is needed too together with the index.yml file.
currently only a local install without a package repo was possible (the "install" section").

I will try to setup an automatic build and publish the packaged chart to the github page of Redis Commander...

**Comment by fulcrum29** [comment]
Any updates on this?
`helm repo add redis-commander https://joeferner.github.io/redis-commander/
Error: looks like "https://joeferner.github.io/redis-commander/" is not a valid chart repository or cannot be reached: failed to fetch https://joeferner.github.io/redis-commander/index.yaml : 404 Not Found`
I still have the same problem :(

**Comment by alexdresko** [comment]
I swear this was working for me a few weeks ago, and now it stopped. So weird.

**Comment by VuiDJi** [comment]
Any news? I have the same issue:
helm repo add redis-commander https://joeferner.github.io/redis-commander/
Error: looks like "https://joeferner.github.io/redis-commander/" is not a valid chart repository or cannot be reached: failed to fetch https://joeferner.github.io/redis-commander/index.yaml : 404 Not Found

**Comment by amigo-akash** [comment]
Does this issue get resolved bcz i still see the same error popping up while adding the repository . Thank you 

 helm repo add redis-commander [](`[[](](https://joeferner.github.io/redis-commander/)`)
)Error: looks like "http://joeferner.github.io/redis-commander" is not a valid chart repository or cannot be reached: failed to fetch http://joeferner.github.io/redis-commander/index.yaml : 404 Not Found
